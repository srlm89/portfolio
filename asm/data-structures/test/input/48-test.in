ERASE actual/48-test.out
INSERT 13 2.364799 The Sniper 720p
PRINT_IN_ORDER actual/48-test.out
INSERT 17 2.860146 Fast and the Furious 1 - 1080p
INSERT 26 5.431928 Crossing Over 720P
REMOVE_ID 13
PRINT_IN_ORDER actual/48-test.out
INSERT 7 9.817548 Duplicity 2009   720p
INSERT 12 5.576158 A.Beautiful.Mind.2001.720p.HDTV.x264-S26
INSERT 30 6.496690 Star Trek [2009] [1080p]
REMOVE_ID 26
PRINT_IN_ORDER actual/48-test.out
INSERT 15 6.800495 Starship Troopers 2  720p
INSERT 19 1.553265 iRobot  720p
INSERT 33 8.512447 Alone in the dark   720p
INSERT 44 7.252524 contents.txt
REMOVE_ID 15
REMOVE_ID 7
PRINT_IN_ORDER actual/48-test.out
INSERT 34 3.164251 Equilibrium 720p
INSERT 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs
INSERT 42 3.500775 The Extreme Movie 720P
INSERT 61 4.332732 Rambo 4 [2008] [720p]
INSERT 57 4.619204 Disaster Movie 2008   720p
REMOVE_ID 19
REMOVE_ID 42
PRINT_IN_ORDER actual/48-test.out
INSERT 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p
INSERT 32 1.471166 Fast and the Furious 3 -  Tokyo Drift - 1080p
INSERT 23 6.351715 Kill Bill Vol 2 [2004]   720p
INSERT 52 0.046442 James Bond 01 - Dr. No [1962] [720p]
INSERT 29 5.421133 James Bond 05 - You Only Live Twice [1967]
INSERT 11 1.618144 Max Payne    720p
REMOVE_ID 32
REMOVE_ID 29
REMOVE_ID 11
PRINT_IN_ORDER actual/48-test.out
INSERT 55 7.550599 Seven Pounds    720p
INSERT 16 1.207331 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN]
INSERT 31 4.463950 Mr. & Mrs. Smith   720p
INSERT 22 0.584563 Independence Day 1080p
INSERT 8 0.072704 City of Embers    720p
INSERT 43 3.963954 Face.Off.1997.720p
INSERT 48 6.300054 Hush   720p
REMOVE_ID 34
REMOVE_ID 44
REMOVE_ID 52
PRINT_IN_ORDER actual/48-test.out
INSERT 49 7.816852 Face.Off.1997.720p
INSERT 35 6.818706 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs
INSERT 40 3.786571 City of Embers    720p
INSERT 5 9.783203 Battlestar Galactica The Plan [720p]
INSERT 10 1.797878 New York I Love You 720p   720p
INSERT 58 0.878624 Babylon A.D    720p
INSERT 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR
INSERT 27 6.154604 Crossing Over 720P
REMOVE_ID 61
REMOVE_ID 16
REMOVE_ID 35
REMOVE_ID 12
PRINT_IN_ORDER actual/48-test.out
INSERT 84 2.677344 Die Hard 1   720p
INSERT 82 9.740163 Lymelife [720p]
INSERT 86 4.543505 Die Hard 2   720p
INSERT 75 1.787183 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE
INSERT 64 0.933707 Surrogates [2009] [720p]
INSERT 73 9.880342 Blade 2   720p
INSERT 4 1.987228 Alien At The Attic 720P
INSERT 59 9.672033 Flash Point    720p
INSERT 67 6.450256 Bee Movie   720p
REMOVE_ID 55
REMOVE_ID 48
REMOVE_ID 57
REMOVE_ID 30
PRINT_IN_ORDER actual/48-test.out
INSERT 76 3.395027 Around The World In 80 Days [2004] [720p]
INSERT 1 0.837949 James Bond 08 - Live and Let Die [1973]
INSERT 71 3.519329 American Psycho   720p
INSERT 46 0.451141 James Bond 11 - MoonRaker [1979]
INSERT 62 6.206886 Pitch Black [720p]
INSERT 70 2.271109 Spiderman 2   1080p
INSERT 90 7.086916 Orphan (2009)    1080p
INSERT 96 9.157865 Gladiator 1080p
INSERT 92 2.230718 Pirates of Caribbean 1-2-3 Collection   1080p
INSERT 94 8.071509 Babel.720p
REMOVE_ID 46
REMOVE_ID 70
REMOVE_ID 96
REMOVE_ID 17
REMOVE_ID 84
PRINT_IN_ORDER actual/48-test.out
INSERT 20 1.370080 X-Men 3   1080p
INSERT 6 4.672647 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV
INSERT 89 2.254992 Tomb Raider Collection (1-2)    720p
INSERT 87 0.756174 007 Licence to Kill [1989] [BluRay] [720p] [DTS]
INSERT 95 3.128815 Underworld Series
INSERT 68 8.608549 Cloverfield   1080p
INSERT 72 9.315589 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE
INSERT 38 4.859994 Land Of the Lost   720p
INSERT 85 8.665109 Big Buck Bunny [1080p]
INSERT 66 2.529125 Ju-on__The_Grudge_(2003)   720p
INSERT 69 6.302081 The Haunting In Connecticut [2009] [720p]
REMOVE_ID 71
REMOVE_ID 5
REMOVE_ID 67
REMOVE_ID 10
REMOVE_ID 49
PRINT_IN_ORDER actual/48-test.out
INSERT 77 5.655453 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 91 0.294852 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p]
INSERT 3 5.620558 Rocky 4 [1985] [BluRay] [720p] [AC3]
INSERT 60 4.767441 enemy at the gates    1080p
INSERT 107 8.928103 Yes Man   720p
INSERT 80 5.763029 Shark Tale [2004] [1080p]
INSERT 47 6.482281 James Bond 21 - Casino Royale [2006]
INSERT 114 9.876398 Star Wars Saga  Collection 720p
INSERT 120 0.332690 Final Destination 2
INSERT 14 4.428857 Night Train   720p
INSERT 50 1.432929 enemy at the gates    1080p
INSERT 54 6.446030 Pirates of Caribbean 1-2-3 Collection   1080p
REMOVE_ID 75
REMOVE_ID 91
REMOVE_ID 58
REMOVE_ID 53
REMOVE_ID 0
REMOVE_ID 86
PRINT_IN_ORDER actual/48-test.out
INSERT 63 5.530099 Lust Caution   720p
INSERT 79 2.479498 Blade 3   720p
INSERT 125 9.713487 SAW III   720p
INSERT 9 7.038925 Golden Compass   720p
INSERT 112 4.137002 Assassination of a High School President 720p
INSERT 36 1.136352 iRobot  720p
INSERT 2 6.847124 9 [720p]
INSERT 45 4.082803 One Missed Call   720p
INSERT 101 5.828153 Meet The Parents    720p
INSERT 100 7.714252 American Psycho   720p
INSERT 133 1.286135 Black Hawk Down   1080p
INSERT 25 6.487317 Bad Boy II [720p]
INSERT 117 4.538006 Pearl Harbor
REMOVE_ID 95
REMOVE_ID 45
REMOVE_ID 66
REMOVE_ID 82
REMOVE_ID 120
REMOVE_ID 69
PRINT_IN_ORDER actual/48-test.out
