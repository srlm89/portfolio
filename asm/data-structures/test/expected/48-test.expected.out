[ 13 2.364799 The Sniper 720p ]
[ 17 2.860146 Fast and the Furious 1 - 1080p ][ 26 5.431928 Crossing Over 720P ]
[ 7 9.817548 Duplicity 2009   720p ][ 12 5.576158 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 30 6.496690 Star Trek [2009] [1080p] ]
[ 12 5.576158 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 19 1.553265 iRobot  720p ][ 30 6.496690 Star Trek [2009] [1080p] ][ 33 8.512447 Alone in the dark   720p ][ 44 7.252524 contents.txt ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 12 5.576158 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 30 6.496690 Star Trek [2009] [1080p] ][ 33 8.512447 Alone in the dark   720p ][ 34 3.164251 Equilibrium 720p ][ 44 7.252524 contents.txt ][ 57 4.619204 Disaster Movie 2008   720p ][ 61 4.332732 Rambo 4 [2008] [720p] ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 12 5.576158 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 30 6.496690 Star Trek [2009] [1080p] ][ 33 8.512447 Alone in the dark   720p ][ 34 3.164251 Equilibrium 720p ][ 44 7.252524 contents.txt ][ 52 0.046442 James Bond 01 - Dr. No [1962] [720p] ][ 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p ][ 57 4.619204 Disaster Movie 2008   720p ][ 61 4.332732 Rambo 4 [2008] [720p] ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 8 0.072704 City of Embers    720p ][ 12 5.576158 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 16 1.207331 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 30 6.496690 Star Trek [2009] [1080p] ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 43 3.963954 Face.Off.1997.720p ][ 48 6.300054 Hush   720p ][ 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p ][ 55 7.550599 Seven Pounds    720p ][ 57 4.619204 Disaster Movie 2008   720p ][ 61 4.332732 Rambo 4 [2008] [720p] ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 5 9.783203 Battlestar Galactica The Plan [720p] ][ 8 0.072704 City of Embers    720p ][ 10 1.797878 New York I Love You 720p   720p ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 27 6.154604 Crossing Over 720P ][ 30 6.496690 Star Trek [2009] [1080p] ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 40 3.786571 City of Embers    720p ][ 43 3.963954 Face.Off.1997.720p ][ 48 6.300054 Hush   720p ][ 49 7.816852 Face.Off.1997.720p ][ 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p ][ 55 7.550599 Seven Pounds    720p ][ 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR ][ 57 4.619204 Disaster Movie 2008   720p ][ 58 0.878624 Babylon A.D    720p ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 4 1.987228 Alien At The Attic 720P ][ 5 9.783203 Battlestar Galactica The Plan [720p] ][ 8 0.072704 City of Embers    720p ][ 10 1.797878 New York I Love You 720p   720p ][ 17 2.860146 Fast and the Furious 1 - 1080p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 27 6.154604 Crossing Over 720P ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 40 3.786571 City of Embers    720p ][ 43 3.963954 Face.Off.1997.720p ][ 49 7.816852 Face.Off.1997.720p ][ 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p ][ 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR ][ 58 0.878624 Babylon A.D    720p ][ 59 9.672033 Flash Point    720p ][ 64 0.933707 Surrogates [2009] [720p] ][ 67 6.450256 Bee Movie   720p ][ 73 9.880342 Blade 2   720p ][ 75 1.787183 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 82 9.740163 Lymelife [720p] ][ 84 2.677344 Die Hard 1   720p ][ 86 4.543505 Die Hard 2   720p ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 1 0.837949 James Bond 08 - Live and Let Die [1973] ][ 4 1.987228 Alien At The Attic 720P ][ 5 9.783203 Battlestar Galactica The Plan [720p] ][ 8 0.072704 City of Embers    720p ][ 10 1.797878 New York I Love You 720p   720p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 27 6.154604 Crossing Over 720P ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 40 3.786571 City of Embers    720p ][ 43 3.963954 Face.Off.1997.720p ][ 49 7.816852 Face.Off.1997.720p ][ 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p ][ 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR ][ 58 0.878624 Babylon A.D    720p ][ 59 9.672033 Flash Point    720p ][ 62 6.206886 Pitch Black [720p] ][ 64 0.933707 Surrogates [2009] [720p] ][ 67 6.450256 Bee Movie   720p ][ 71 3.519329 American Psycho   720p ][ 73 9.880342 Blade 2   720p ][ 75 1.787183 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 76 3.395027 Around The World In 80 Days [2004] [720p] ][ 82 9.740163 Lymelife [720p] ][ 86 4.543505 Die Hard 2   720p ][ 90 7.086916 Orphan (2009)    1080p ][ 92 2.230718 Pirates of Caribbean 1-2-3 Collection   1080p ][ 94 8.071509 Babel.720p ]
[ 0 9.836045 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 1 0.837949 James Bond 08 - Live and Let Die [1973] ][ 4 1.987228 Alien At The Attic 720P ][ 6 4.672647 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 8 0.072704 City of Embers    720p ][ 20 1.370080 X-Men 3   1080p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 27 6.154604 Crossing Over 720P ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 38 4.859994 Land Of the Lost   720p ][ 40 3.786571 City of Embers    720p ][ 43 3.963954 Face.Off.1997.720p ][ 53 4.649097 Fast and the Furious 3 -  Tokyo Drift - 1080p ][ 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR ][ 58 0.878624 Babylon A.D    720p ][ 59 9.672033 Flash Point    720p ][ 62 6.206886 Pitch Black [720p] ][ 64 0.933707 Surrogates [2009] [720p] ][ 66 2.529125 Ju-on__The_Grudge_(2003)   720p ][ 68 8.608549 Cloverfield   1080p ][ 69 6.302081 The Haunting In Connecticut [2009] [720p] ][ 72 9.315589 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 73 9.880342 Blade 2   720p ][ 75 1.787183 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 76 3.395027 Around The World In 80 Days [2004] [720p] ][ 82 9.740163 Lymelife [720p] ][ 85 8.665109 Big Buck Bunny [1080p] ][ 86 4.543505 Die Hard 2   720p ][ 87 0.756174 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 89 2.254992 Tomb Raider Collection (1-2)    720p ][ 90 7.086916 Orphan (2009)    1080p ][ 92 2.230718 Pirates of Caribbean 1-2-3 Collection   1080p ][ 94 8.071509 Babel.720p ][ 95 3.128815 Underworld Series ]
[ 1 0.837949 James Bond 08 - Live and Let Die [1973] ][ 3 5.620558 Rocky 4 [1985] [BluRay] [720p] [AC3] ][ 4 1.987228 Alien At The Attic 720P ][ 6 4.672647 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 8 0.072704 City of Embers    720p ][ 14 4.428857 Night Train   720p ][ 20 1.370080 X-Men 3   1080p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 27 6.154604 Crossing Over 720P ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 38 4.859994 Land Of the Lost   720p ][ 40 3.786571 City of Embers    720p ][ 43 3.963954 Face.Off.1997.720p ][ 47 6.482281 James Bond 21 - Casino Royale [2006] ][ 50 1.432929 enemy at the gates    1080p ][ 54 6.446030 Pirates of Caribbean 1-2-3 Collection   1080p ][ 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR ][ 59 9.672033 Flash Point    720p ][ 60 4.767441 enemy at the gates    1080p ][ 62 6.206886 Pitch Black [720p] ][ 64 0.933707 Surrogates [2009] [720p] ][ 66 2.529125 Ju-on__The_Grudge_(2003)   720p ][ 68 8.608549 Cloverfield   1080p ][ 69 6.302081 The Haunting In Connecticut [2009] [720p] ][ 72 9.315589 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 73 9.880342 Blade 2   720p ][ 76 3.395027 Around The World In 80 Days [2004] [720p] ][ 77 5.655453 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 80 5.763029 Shark Tale [2004] [1080p] ][ 82 9.740163 Lymelife [720p] ][ 85 8.665109 Big Buck Bunny [1080p] ][ 87 0.756174 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 89 2.254992 Tomb Raider Collection (1-2)    720p ][ 90 7.086916 Orphan (2009)    1080p ][ 92 2.230718 Pirates of Caribbean 1-2-3 Collection   1080p ][ 94 8.071509 Babel.720p ][ 95 3.128815 Underworld Series ][ 107 8.928103 Yes Man   720p ][ 114 9.876398 Star Wars Saga  Collection 720p ][ 120 0.332690 Final Destination 2 ]
[ 1 0.837949 James Bond 08 - Live and Let Die [1973] ][ 2 6.847124 9 [720p] ][ 3 5.620558 Rocky 4 [1985] [BluRay] [720p] [AC3] ][ 4 1.987228 Alien At The Attic 720P ][ 6 4.672647 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 8 0.072704 City of Embers    720p ][ 9 7.038925 Golden Compass   720p ][ 14 4.428857 Night Train   720p ][ 20 1.370080 X-Men 3   1080p ][ 22 0.584563 Independence Day 1080p ][ 23 6.351715 Kill Bill Vol 2 [2004]   720p ][ 25 6.487317 Bad Boy II [720p] ][ 27 6.154604 Crossing Over 720P ][ 31 4.463950 Mr. & Mrs. Smith   720p ][ 33 8.512447 Alone in the dark   720p ][ 36 1.136352 iRobot  720p ][ 38 4.859994 Land Of the Lost   720p ][ 40 3.786571 City of Embers    720p ][ 43 3.963954 Face.Off.1997.720p ][ 47 6.482281 James Bond 21 - Casino Royale [2006] ][ 50 1.432929 enemy at the gates    1080p ][ 54 6.446030 Pirates of Caribbean 1-2-3 Collection   1080p ][ 56 7.401304 Terminator.1984.720p.BluRay.x264-ESiR ][ 59 9.672033 Flash Point    720p ][ 60 4.767441 enemy at the gates    1080p ][ 62 6.206886 Pitch Black [720p] ][ 63 5.530099 Lust Caution   720p ][ 64 0.933707 Surrogates [2009] [720p] ][ 68 8.608549 Cloverfield   1080p ][ 72 9.315589 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 73 9.880342 Blade 2   720p ][ 76 3.395027 Around The World In 80 Days [2004] [720p] ][ 77 5.655453 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 79 2.479498 Blade 3   720p ][ 80 5.763029 Shark Tale [2004] [1080p] ][ 85 8.665109 Big Buck Bunny [1080p] ][ 87 0.756174 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 89 2.254992 Tomb Raider Collection (1-2)    720p ][ 90 7.086916 Orphan (2009)    1080p ][ 92 2.230718 Pirates of Caribbean 1-2-3 Collection   1080p ][ 94 8.071509 Babel.720p ][ 100 7.714252 American Psycho   720p ][ 101 5.828153 Meet The Parents    720p ][ 107 8.928103 Yes Man   720p ][ 112 4.137002 Assassination of a High School President 720p ][ 114 9.876398 Star Wars Saga  Collection 720p ][ 117 4.538006 Pearl Harbor ][ 125 9.713487 SAW III   720p ][ 133 1.286135 Black Hawk Down   1080p ]
